router/pkg/config/config.go

```diff
@@ -783,6 +784,10 @@ type ApolloCompatibilityReplaceInvalidVarErrors struct {
 	Enabled bool `yaml:"enabled" envDefault:"false" env:"APOLLO_COMPATIBILITY_REPLACE_INVALID_VAR_ERRORS_ENABLED"`
 }
 
+type ApolloCompatibilityReplaceInvalidVarErrorStatus struct {
+	Enabled bool `yaml:"enabled" envDefault:"false" env:"APOLLO_COMPATIBILITY_REPLACE_INVALID_VAR_ERROR_STATUS_ENABLED"`
```

Comment:
this needs to be added to the json schema

Reply:
Added. Please check.